apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -301,16 +287,15 @@ export class SendMessage {
   }
 
   @Instrument()
-  private async filterPreferredChannels(job: JobEntity): Promise<boolean> {
+  private async filterPreferredChannels(command: SendMessageCommand): Promise<boolean> {
+    const { job } = command;
+
     const template = await this.getNotificationTemplate({
```

Comment:
If you dont mind could you rename this one if we are already here 🫠
I really hate those `template` variables in the code. :( 

```suggestion
    const workflow = await this.getWorkflow({
```

Reply:
Good catch, done!